src/string_utils.cpp

```diff
@@ -71,6 +71,31 @@ vector<std::string> split(const std::string& in)
   return out;
 }
 
+vector<std::string> split(const std::string& in, char delim)
```

Comment:
Now that we're back to space-delimited arrays in the XML file, I think this function can be returned to its original state.